exec/java-exec/src/main/resources/rest/index.ftl

```diff
@@ -79,9 +91,14 @@
                   </span>
                 </td>
                 <td id="status" >${drillbit.getState()}</td>
+                <!-- #if (model.shouldShowAdminInfo() || !model.isAuthEnabled()) -->
```

Comment:
Please remove the comment.

Reply:
Missed catching that without an IDEA. Thanks